packages/library-base/src/commonMain/kotlin/io/realm/RealmList.kt

```diff
@@ -37,19 +37,19 @@ import kotlinx.coroutines.flow.Flow
  * that inject values into a class. Unmanaged elements in a list can be added to a Realm using the
  * [MutableRealm.copyToRealm] method.
  */
-interface RealmList<E> : MutableList<E>, Flowable<RealmList<E>> {
+interface RealmList<E> : MutableList<E> {
 
     /**
-     * Observes changes to the RealmList. If there is any change to the list, the flow will emit the
-     * updated RealmResult. The flow will continue running indefinitely until canceled or until the
-     * parent object is deleted.
+     * Observes changes to the RealmList. The flow will emit a [ListChange] once subscribed, and then
```

Comment:
Should this be `The flow will emit a `InitialListChange` once subscribed, and then a `UpdatedList` on every change to the list` ?

Reply:
Yes, much better